cmd/argocd/commands/bcrypt.go

```diff
@@ -16,6 +16,8 @@ func NewBcryptCmd() *cobra.Command {
 	var bcryptCmd = &cobra.Command{
 		Use:   "bcrypt",
 		Short: "Generate bcrypt hash for the admin password",
+		Example: `# Generate bcrypt hash for the admin password 
```

Comment:
I'd remove the word "admin." The command can be used to generate a hash for any password, not just the admin password.